src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -136,10 +149,51 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
     this.fetchIstioObjectDetailsFromProps(this.props.match.params);
   };
 
-  fetchIstioObjectDetailsFromProps = (props: IstioConfigId) => {
-    const promiseConfigDetails = props.objectSubtype
+  newIstioObjectPromise = (props: IstioConfigId, validate: boolean) => {
+    return props.objectSubtype
       ? API.getIstioConfigDetailSubtype(props.namespace, props.objectType, props.objectSubtype, props.object)
-      : API.getIstioConfigDetail(props.namespace, props.objectType, props.object, true);
+      : API.getIstioConfigDetail(props.namespace, props.objectType, props.object, validate);
+  };
+
+  fetchIstioObjectDetailsFromProps = (props: IstioConfigId) => {
+    const promiseConfigDetails = this.newIstioObjectPromise(props, true);
+
+    window.clearInterval(this.timerId);
+    this.timerId = window.setInterval(() => {
+      const timerPromise = this.newIstioObjectPromise(props, false);
+      timerPromise
+        .then(resultConfigDetails => {
+          if (resultConfigDetails.data && this.state.originalIstioObjectDetails) {
+            const [changed, type, newResourceVersion] = compareResourceVersion(
+              this.state.originalIstioObjectDetails,
+              resultConfigDetails.data
+            );
+            if (changed) {
+              AlertUtils.addWarning(
+                'Istio object type [' +
+                  type +
+                  '] name [' +
+                  props.object +
+                  '] has been modified externally to Kiali. New ResourceVersion:  ' +
+                  newResourceVersion
+              );
```

Comment:
nitpick: The message is shown even if the user has another tab opened and changed the instance there. Meaning that the change has happened within Kiali console. 
Therefore, I'd put a wider message covering this scenario too. Something like 'sidecar:no-ip-tables has a newer version (9182178). Reload to see the new version.'

Reply:
+1 true